### PR TITLE
Fix Bug 1199447, add new promo for thimble

### PR DESCRIPTION
**Pull request by schalkneethling**
Still waiting on some feedback from the bug so, marking as WIP and DNM with l10n

### CODE REVIEW

**Comment by schalkneethling** [comment]
@alexgibson @flodolo r?

File: bedrock/mozorg/templates/mozorg/home/home.html [hunk 1]
```diff
@@ -185,7 +185,15 @@ <h2 class="primary go">{{ _('Teach the Web') }}</h2>
         <li id="promo-7" class="item promo-face">
           {{ promo_face(3) }}
         </li>
-      {% if l10n_has_tag('webmaker_promo_introducing') %}
+      {% if l10n_has_tag('webmaker_promo_thimble') %}
+        <li id="promo-8" class="item promo-small-landscape thimble" tabindex="0" data-name="Thimble">
+          <a class="panel-link" rel="external" href="https://thimble.mozilla.org/">
+            <div class="primary">
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/home/home.html]: I don't think we need this `<div class="primary">`?

File: bedrock/mozorg/templates/mozorg/home/home.html [hunk 2]
```diff
@@ -185,7 +185,15 @@ <h2 class="primary go">{{ _('Teach the Web') }}</h2>
...
+              <h2 class="go">{{ _('Try the Thimble code editor') }}</h2>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/home/home.html]: Not sure we need `class="go"` her either?

**Comment by schalkneethling** [comment]
Nope, they were not needed. \o/ for less HTML and CSS

File: media/css/mozorg/home/home-promo.less [hunk 3]
```diff
@@ -890,6 +890,29 @@ html[lang|="en"] {
...
+/***************************************************************************/
+// Thimble
+.promo-small-landscape.thimble {
+
+    background-color: #2D9961;
+
+    h2 {
+        .at2x('/media/img/home/panel/thimble/mozilla-thimble.png', 69px, 75px);
+        background-repeat: no-repeat;
+        margin: 30px 20px;
```
Comment (alexgibson) [inline media/css/mozorg/home/home-promo.less]: Nit: slightly nicer centering with `25px @baseLine`?

File: media/css/mozorg/home/home-promo.less [hunk 4]
```diff
@@ -890,6 +890,29 @@ html[lang|="en"] {
...
+        padding: 8px 0 0 85px;
+        height: 75px;
+    }
+}
+
+.html-rtl {
+    h2 {
```
Comment (alexgibson) [inline media/css/mozorg/home/home-promo.less]: Overly specific selector - should this be `.promo-small-landscape.thimble h2`?

**Comment by alexgibson** [comment]
I would suggest we give non-english locales as much room for text as possible, similarly to how we handle the other small promos? https://github.com/mozilla/bedrock/blob/master/media/css/mozorg/home/home-promo.less#L756

File: media/css/mozorg/home/home-promo.less (hunk 4 above)
Comment (schalkneethling) [inline media/css/mozorg/home/home-promo.less]: Hah, I had it like that and then changed it. Totally agree, targeting all h2 when it is rtl is not a good idea ;p

**Comment by schalkneethling** [comment]
Ok, I did some tweaks and testing. Tested with German (below), Italian and French, and the spacing looks good.

screen shot 2015-09-01 at 16 20 49

Also tested RTL using Arabic.

screen shot 2015-09-01 at 16 24 20

**Comment by schalkneethling** [comment]
Ok, the spacing looks better now after a small tweak for the on EN languages.

**Comment by alexgibson** [comment]
r+ on the code changes - care to squash?

I think Flod might be on PTO, so pinging @pascalchevrel to see if we're ok to merge this. We're using an l10n tag so it should be fine, but we need to make sure folks are aware there's a new string.

**Pull request merged**